### PR TITLE
Allow user clicks on confirm dialog buttons

### LINKED ISSUE #990: Unpublish ditributed post
### Describe your question

Distributor 1.9.0
WP 6.1.1

Created a post on source/original site, and distribute to destination site. Publish it on destination site, and attempt to switch back to draft. The modal is disabled

Screenshot 2023-01-04 at 09 41 44

How can I unpublish a distributed post?

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by av3nger**
### Description of the Change
This change prevents locking out the buttons in Gutenberg confirmation dialogs, by reverting the `opacity`, `pointer-events` and `cursor` values set for distributed posts.
Closes #990 

### How to test the Change
Created a post on source/original site, and distribute to the destination site. When publishing to the destination site, unchecking the "Save as Draft" option. Attempt to switch back to draft on the destination site - confirmation modal buttons are disabled. After applying this PR, the modal buttons should be enabled.

### Changelog Entry
Fixed - Allow clicking on the `Save to draft` confirmation modal buttons

### Credits
Props @av3nger 

### Checklist:

- [x] I agree to follow this project's [**Code of Conduct**](https://github.com/10up/.github/blob/trunk/CODE_OF_CONDUCT.md).
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my change.
- [x] All new and existing tests pass.

### CODE REVIEW

**Review by peterwilsoncc — COMMENTED** [review]
Thanks Anton,

This is working in WP 6.2, I'm just considering a few things:

* the `confirmDialog` component was introduced in WP 6.0, so for older versions the CSS will probably need to target the underlying `Model` component's classes.
* whether the CSS only change is too permissive, is it possible to only allow the pointer events if the popover is shown when switch to draft is clicked? (Looking at `PostSwitchToDraftButton` I suspect the answer is no, but you may know better than I.)

Let me ponder this for a little longer.

**Comment by av3nger** [comment]
@peterwilsoncc, in pre 6.0 moving from Draft to Published state is done via the "Publish" button. Doesn't look like it's disabled on my tests

File: assets/css/gutenberg-syndicated-post.scss [hunk 1]
```diff
@@ -40,6 +47,7 @@ body.dt-linked-post {
 
 	.components-panel__body-toggle,
 	.edit-post-post-visibility__toggle,
+	.editor-post-switch-to-draft,
```
Comment (peterwilsoncc) [inline assets/css/gutenberg-syndicated-post.scss:50]: @av3nger As the main issue in #990 was allowing users to move a distributed copy of a post back to draft, I've pushed this to your PR to account for UI changes in WP 6.3.

**Review by peterwilsoncc — APPROVED** [review]
Thanks Anton, this LGTM.

Testing notes: tested converting a distributed post back to draft in WP 5.7 onwards.

**Pull request merged**